tests/ui/numbers-arithmetic/issue-109567.rs

```diff
@@ -1,11 +1,12 @@
 // run-pass
 // check-run-results
+#![feature(const_fn_floating_point_arithmetic)]
 
-pub fn f() -> f64 {
+fn f() -> f64 {
     std::hint::black_box(-1.0) % std::hint::black_box(-1.0)
 }
 
-pub fn g() -> f64 {
+const fn g() -> f64 {
     -1.0 % -1.0
 }
```

Comment:
This does nothing. Making a `const fn` just means the function can be called from a `const`, but you are not doing that.

What I meant is something like
```rust
const G: f64 = -1.0 % -1.0;
```
Then you also don't need the feature gate.

Reply:
Fixed it.
According to https://github.com/rust-lang/rust/pull/109573#issuecomment-1483685689, it's not suggested to make changes to this file? 🤔